# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gdk-pixbuf/no-tests.patch
# Copyright (C) 2019 - 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Index: meson.build
--- ./meson.build.orig
+++ ./meson.build
@@ -398,7 +398,7 @@ endif
 subdir('po')
 
 if not meson.is_cross_build()
-  subdir('tests')
+  #subdir('tests')
   subdir('thumbnailer')
 endif
